kuma/users/tests/test_views.py

```diff
@@ -1602,3 +1602,16 @@ def test_invalid_uid_fails(wiki_user, client):
     assert response.status_code == 200
     assert_no_cache_header(response)
     assert b'This link is no longer valid.' in response.content
+
+
+def test_signin_landing(db, client, settings):
+    settings.MULTI_AUTH_ENABLED = True
+    response = client.get(reverse('socialaccount_signin'))
+    assert response.status_code == 200
+    doc = pq(response.content)
+    assert 'Create your MDN Web Docs Account' in doc.find('h1').text()
+
+
+def test_signin_landing_multi_auth_disabled(db, client):
```

Comment:
At some point soon, we're going to switch the default of `MULTI_AUTH_ENABLED` to true. It's false right now just because of timing and us being careful. 
So, I'd put in a `settings.MULTI_AUTH_ENABLED = False` on the first line. It reads better that way.